tests/PrependInlineIndexTest.php

```diff
@@ -0,0 +1,13 @@
+<?php
+namespace nadar\quill\tests;
+
+class PrependInlineIndexTest extends DeltaTestCase
+{
+    public $json = <<<'JSON'
+{"ops":[{"attributes":{"italic":true},"insert":"italic"},{"attributes":{"bold":true,"link":"https://nadar.io"},"insert":"link"},{"insert":"\n"}]}
```

Comment:
I think it is nice to format these with each operation on its own line.

Reply:
sure. I just used the output.php "generate unit test" button. We could format the output there already.